routesrv/eskipbytes.go

```diff
@@ -50,12 +52,15 @@ var (
 // provides synchronized r/w access to them. Additionally it can
 // serve as an HTTP handler exposing its content.
 type eskipBytes struct {
+	mu           sync.RWMutex
 	data         []byte
-	etag         string
```

Comment:
Why we don't need etag more?

Reply:
I think it was just renamed to `hash` but I think `etag` is more expressive, on another note if we will rename here it makes sense to rename in https://github.com/zalando/skipper/blob/master/eskipfile/remote.go#L31

Reply:
gzip-encoded content must have different Etag from plain text content. I derive gzip Etag by adding `+gzip` suffix (Apache's mod_deflate adds `-gzip` but I saw `+gzip` from `curl -v --compressed example.com` and I like it more with plus sign).
Since Etag value must be quoted I store "hash" and quote the value before setting Etag header therefore I renamed field member from etag to hash. I.e. `hash == "cafebabe"` translates into `Etag: "cafebabe"` or `Etag: "cafebabe+gzip"` header values depending on the Accept-Encoding.

Reply:
> if we will rename here it makes sense to rename in

Not, there is it a Etag value (quoted hash)